Cinema-System-ESD-Component B/UWEFlix_django/uweflix/templates/uweflix/thanks.html

```diff
@@ -10,6 +10,9 @@ <h1>Thank you</h1>
 <h3>Your order number is #{{ request.session.transaction }}</h3>
 <p>You're going to see: {{ request.session.film }}</p>
 <p>In Screen {{ request.session.screen }} on {{ request.session.date }} at {{ request.session.time }}</p>
+{% if request.session.covid_restrictions%}
+<p>As this is a Covid-19 restricted showing, your seat has been allocated to {{request.session.allocated_seat}} for you. </p>
```

Comment:
Minor/Personal opinion : Might be worth making the <p> tag more concise, up to you.

Reply:
I've updated it now to be more readable